eventcollectors/src/main/kotlin/events/boudicca/eventcollector/collectors/StiftskonzerteCollector.kt

```diff
@@ -0,0 +1,65 @@
+package events.boudicca.eventcollector.collectors
+
+import events.boudicca.SemanticKeys
+import events.boudicca.api.eventcollector.Event
+import events.boudicca.api.eventcollector.Fetcher
+import events.boudicca.api.eventcollector.TwoStepEventCollector
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Element
+import java.time.LocalDate
+import java.time.LocalTime
+import java.time.OffsetDateTime
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
+
+class StiftskonzerteCollector : TwoStepEventCollector<String>("stiftskonzerte") {
+
+    private val fetcher = Fetcher(2000)
+
+    override fun getAllUnparsedEvents(): List<String> {
+        val document = Jsoup.parse(fetcher.fetchUrl("https://www.stiftskonzerte.at/programm-und-karten/"))
+        return document.select("div.entry-footer-links a").not("a.open-modal")
+            .map { it.attr("href") }
+    }
+
+    override fun parseEvent(event: String): Event {
+        val eventSite = Jsoup.parse(fetcher.fetchUrl(event))
+        val name = eventSite.select("header.entry-header h1").text()
+
+        val data = mutableMapOf<String, String>()
+        data[SemanticKeys.URL] = event
+        data[SemanticKeys.DESCRIPTION] = eventSite.select("div.entry-flexible-content").text()
+
+        val locationTimeDiv = eventSite.select("div.entry-content div.location")
+        locationTimeDiv.select("br").after("\\n")
```

Comment:
does this line actually do something?

Reply:
yes, it helps to preserve the line breaks by adding \n after each br tag, because the br tags disappear when using .text() later

Reply:
ohh, awesome! thanks for explaining